core/client/src/main/java/alluxio/client/file/FileSystemMasterClient.java

```diff
@@ -248,20 +247,6 @@ public Long call() throws AlluxioTException, TException {
   }
 
   /**
-   * @return the under file system address
-   * @throws ConnectionFailedException if network connection failed
-   * @throws IOException if an I/O error occurs
-   */
-  public synchronized String getUfsAddress() throws IOException, ConnectionFailedException {
```

Comment:
this is an internal API so I removed it along with all internal client-side dependencies ... the externally visible API is the thrift and REST server-side endpoints, which I marked deprecated